### ISSUE TITLE
Object Reference error on Deleted group in Collection view

### ISSUE THREAD

**Issue by Scrxtchy** [japanesemediamanager/jmmclient#467]
Directly related to my issue japanesemediamanager/jmmserver#430
Upon displaying a group under the affect of a _deleted_ series, JMM Desktop would display an object reference issue.
This is after pressing `Delete Series` from the group's options in the collection screen.
The entire view is refreshed, but the series remains in the collection, which when prompted displays the error
![](https://r.kyaa.sg/xaaiub.png)

```
[2016-09-22 15:59:31:367] Info|EpisodeList.PopulateToolbars => AnimeSeriesVM.get_EpisodeTypes => AnimeSeriesVM.RefreshEpisodes Got episode data from service: Fate/Zero in 0.4922 ms
[2016-09-22 15:59:31:400] Info|EpisodeList.PopulateToolbars => AnimeSeriesVM.get_EpisodeTypes => AnimeSeriesVM.RefreshEpisodes Got episode contracts: Fate/Zero in 27.5483 ms
[2016-09-22 15:59:31:400] Info|EpisodeList.PopulateToolbars => AnimeSeriesVM.get_EpisodeTypes => AnimeSeriesVM.RefreshEpisodes Sorted episode contracts: Fate/Zero in 0 ms
[2016-09-22 16:00:36:796] Error|RoutedEventArgs.InvokeHandler => MainWindow.lbGroupsSeries_SelectionChanged => Utils.ShowErrorMessage System.NullReferenceException: Object reference not set to an instance of an object.
   at JMMClient.AnimeSeriesVM.Populate(Contract_AnimeSeries contract) in D:\My Files\Documents\GitHub\jmmclient\JMMClient\ViewModel\AnimeSeriesVM.cs:line 785
   at JMMClient.AnimeSeriesVM.RefreshBase() in D:\My Files\Documents\GitHub\jmmclient\JMMClient\ViewModel\AnimeSeriesVM.cs:line 866
   at JMMClient.MainWindow.lbGroupsSeries_SelectionChanged(Object sender, SelectionChangedEventArgs e) in D:\My Files\Documents\GitHub\jmmclient\JMMClient\MainWindow.xaml.cs:line 3386
[2016-09-22 16:14:24:172] Trace|MainWindow.CheckForUpdatesNew => JMMAutoUpdatesHelper.GetLatestVersionInfo => Utils.DownloadWebPage DownloadWebPage: http://www.jmediamanager.org/latestdownloads/versions.xml
[2016-09-22 16:14:26:998] Trace|BackgroundWorker.OnDoWork => MainWindow.showDashboardWorker_DoWork => DashboardVM.RefreshData Dashboard Time: RefreshGroupsSeriesData: 2312.6029
[2016-09-22 16:14:29:182] Warn|UTorrentHelperVM.Init => UTorrentHelper.Init => UTorrentHelper.PopulateToken Credentials are not valid for uTorrent
[2016-09-22 16:14:34:961] Info|EpisodeList.PopulateToolbars => AnimeSeriesVM.get_EpisodeTypes => AnimeSeriesVM.RefreshEpisodes Got episode data from service: Fate/Stay Night in 3.5062 ms
[2016-09-22 16:14:34:996] Info|EpisodeList.PopulateToolbars => AnimeSeriesVM.get_EpisodeTypes => AnimeSeriesVM.RefreshEpisodes Got episode contracts: Fate/Stay Night in 30.0536 ms
[2016-09-22 16:14:34:996] Info|EpisodeList.PopulateToolbars => AnimeSeriesVM.get_EpisodeTypes => AnimeSeriesVM.RefreshEpisodes Sorted episode contracts: Fate/Stay Night in 0 ms
[2016-09-22 16:22:43:673] Error|RoutedEventArgs.InvokeHandler => MainWindow.lbGroupsSeries_SelectionChanged => Utils.ShowErrorMessage System.NullReferenceException: Object reference not set to an instance of an object.
   at JMMClient.AnimeSeriesVM.Populate(Contract_AnimeSeries contract) in D:\My Files\Documents\GitHub\jmmclient\JMMClient\ViewModel\AnimeSeriesVM.cs:line 785
   at JMMClient.AnimeSeriesVM.RefreshBase() in D:\My Files\Documents\GitHub\jmmclient\JMMClient\ViewModel\AnimeSeriesVM.cs:line 866
   at JMMClient.MainWindow.lbGroupsSeries_SelectionChanged(Object sender, SelectionChangedEventArgs e) in D:\My Files\Documents\GitHub\jmmclient\JMMClient\MainWindow.xaml.cs:line 3386
[2016-09-22 16:28:06:841] Trace|BackgroundWorker.OnDoWork => MainWindow.showDashboardWorker_DoWork => DashboardVM.RefreshData Dashboard Time: RefreshGroupsSeriesData: 9.5175
```

**Comment by ElementalCrisis** [comment]
I'm not having this issue on 3.7, will continue to investigate but it might have been fixed with changes made in 3.7.

**Comment by Scrxtchy** [comment]
Here's a pretty bad recording on what's happening my end, if it may help
https://youtu.be/VxwocscPI5U

**Comment by ElementalCrisis** [comment]
Yeah seems like its been fixed in 3.7, we'll be having a public beta in a couple of days where you can try and if you can replicate we'll look into it.

**Comment by Scrxtchy** [comment]
Unable to replicate this in 3.7. If caught again, will reopen
